### PR TITLE
Portfolio finish mobile version

### PR DESCRIPTION
**Pull request by djo1975**
I added index.html and style.css and folder with images and another one with linters....

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+  padding: 0;
+  background-color: #e5e5e5;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+}
+
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure your page fits the entire screen size, The user doesn't need to scroll to the right to see the whole page.
`Kindly you may consider giving you page of 100%`

Your design :point_down: 
image
image


Figma template :point_down: 
image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+  cursor: pointer;
+  user-select: none;
+}
+
+h1 {
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
```
Comment (christianbiring1) [inline style.css:45]: - [ ] Kindly consider sticking to the original design as much as possible. Your typos(font), color, and spacing(margin, padding) are supposed to match with the Figma design.
`To get a specific element style, Kindly make sure you click the element and you will see the styles applied to it on the right side in Figma` :smile: 

Your design :point_down: 
image

Figma template :point_down: 
image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+  color: #000;
+}
+
+button {
+  border: 1px solid purple;
+  border-radius: 5px;
+  padding: 5px 20px;
+  background-color: white;
+  color: purple;
+  cursor: pointer;
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure you stick to the original design provided in Figma. Your buttons and stack list need to be similar to the ones in the design. please fix that.
`Hint: Ti get an element applied styles, Kindly click that element and you will see the styles popup on the right side in Figma`

Your design :point_down: 
image

Figma template :point_down: 
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+                    </ul>
+                  </div>
+        <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+        
+                  <ul class="prog-lang">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                  </ul>
+                  <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
```
Comment (christianbiring1) [inline index.html:111]: [OPTIONAL]
- [ ] It would be better to use a button here instead of an anchor tag

File: index.html [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+                      </ul>
+                    </div>
+          
+                    <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+                    <ul class="prog-lang">
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javaScript</li>
+                    </ul>
+                    <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
```
Comment (christianbiring1) [inline index.html]: [OPTIONAL]
- [ ] It would be better to use a button here instead of an anchor tag

File: index.html [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
```
Comment (christianbiring1) [inline index.html]: [OPTIONAL]
- [ ] It would be better to use a button here instead of an anchor tag

File: style.css [hunk 7]
```diff
@@ -0,0 +1,332 @@
...
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+p {
+  width: 327px;
+  height: 146px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
```
Comment (christianbiring1) [inline style.css]: - [ ] As you are using `Poppins` as font family, Kindly make sure you have imported from Google fonts at the top of your CSS or in your HTML file in the header section

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @djo1975 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commit messages :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  [ ] Being more descriptive in your README. Kindly make sure you always fill in the important fields from the original template that applies to your project.
`You may consider adding the section: Get started and all the ones applied to your project.`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by djo1975** [comment]
can someone help me with this

File: style.css [hunk 8]
```diff
@@ -0,0 +1,316 @@
...
+  color: #000;
+}
+
+button {
+  border: 1px solid purple;
+  border-radius: 5px;
+  padding: 5px 20px;
+  background-color: white;
+  color: purple;
+  cursor: pointer;
```
Comment (andgarzonmal) [inline style.css]: - [ ]  As mentioned by the previous reviewer please add a blue border to the button in order to stick to the Figma design.

## Your design 

image

## Figma design
 
image

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @djo1975✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by RiyaBulia12 — APPROVED** [review]
## STATUS: APPROVED 🏆🥇 

### Hi @djo1975  👋🏽  

Your project is complete! Great Job 🎉 🏆 🍹  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 🎉 🥳 🥳 



## Highlights
* No Linter errors ✔️
* Good commit messages ✔️
* Professional Readme file ✔️
* Descriptive PR ✔️
* GitHub flow used ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ **OPTIONAL** ] I think it would be better to have an elaborative description for PR which helps visitors/viewers to get an idea of what they are looking at  😄 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**